src/app/Component/RegisterUser/RegisterUser.js

```diff
@@ -80,20 +84,23 @@ class Registration extends Component {
             }
             else {
                 delete this.errors.password;
-                this.errors.cpassword = 'Password field and confirm passoword should match';
```

Comment:
Change the variable name as cpassword. Please keep a meaningful names.so the other persons can understand